Sources/Core/Public/GTMSessionFetcher/GTMSessionFetcher.h

```diff
@@ -1100,9 +1101,12 @@ __deprecated_msg("implement GTMSessionFetcherAuthorizer instead")
 @property(atomic, readonly, getter=isFetching) BOOL fetching;
 
 // Cancel the fetch of the request that's currently in progress.  The completion handler
-// will not be called.
+// will be called if the property `stopFetchingTriggersCallbacks` is `YES`.
 - (void)stopFetching;
 
+// Call callbacks in `stopFetching`.
+@property(atomic, assign) BOOL stopFetchingTriggersCallbacks;
```

Comment:
Should this be limited to the completion handler?

`stopFetchingTriggersCompletionHandler` ?